backend/seed_data/test_cases.yaml

```
@@ -86,4 +86,33 @@ test_cases:
           - "[3,1,4,None,2]"
       - input: "[5]"  
         outputs:
-          - "[5]"
\ No newline at end of file
+          - "[5]"
+
+# Strings and Arrays
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)